app/routes/create-catch-certificate/$catchCertificate/add-your-reference/index.tsx

```diff
@@ -63,8 +63,8 @@ export const action: ActionFunction = async ({
 };
 
 const UserReferencePage = () => {
-  const { errors = {} } = useActionData() || {};
-
+  const { errors = {}, userReference } = useActionData() || {};
```

Comment:
@daniele-zurico are we able to add a type here? I'm just curious I'm not entirely sure myself

Reply:
the only way let me do it is:
```tsx
const { errors = {}, userReference } = useActionData() || {} as IUserReference;
```
but I'm not sure is correct